deploy/static-pods/addon-manager.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-addon-manager
+  namespace: kube-system
+  version: v2
+spec:
+  hostNetwork: true
```

Comment:
Could you drop these yaml files from the PR for now? I tried this locally and there's a bug in localkube that prevents the addon manager from working now because it uses host-networking. I'll have a PR soon to fix it, but for now this will generate a bunch of error log lines.